src/blackcore/vatsim/vatsimserverfilereader.h

```diff
@@ -50,7 +50,7 @@ namespace BlackCore::Vatsim
 
     signals:
         //! Data have been read
-        void dataFileRead(int kB);
+        void dataFileRead(double kB);
```

Comment:
Why change the parameter type to `double kB` rather than `int bytes`? Seems like that would be simpler.

Reply:
Changed